collections/motoko/src/btreemap_order_128.mo

```diff
@@ -0,0 +1,48 @@
+import Map "mo:btreemap/BTree";
+import Nat32 "mo:base/Nat32";
+import Iter "mo:base/Iter";
+import Option "mo:base/Option";
+import Random "random";
+
+actor {
```

Comment:
Can we make this as an `actor class` that takes `order` as init args? Then we can tune the number in `perf.sh`.

Reply:
I decided to just update the BTree to default to an order of 32 after some internal testing.